tests/component-tests/textfieldSlider/js/TextfieldSliderTests.js

```diff
@@ -99,8 +183,26 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                     value: 1
```

Comment:
@jobara - The expected values in the `testInputField` calls when testing invalid values, you mean?

They use direct values in the current test code (I'm guessing because of the 'pass null, expect 0' test):
https://github.com/fluid-project/infusion/blob/master/tests/component-tests/textfieldSlider/js/TextfieldSliderTests.js#L102
https://github.com/fluid-project/infusion/blob/master/tests/component-tests/textfieldSlider/js/TextfieldSliderTests.js#L103